src/components/GitmojiList/index.js

```diff
@@ -29,6 +30,13 @@ const GitmojiList = (props: Props): Element<'div'> => {
       })
     : props.gitmojis
 
+  const { query } = useRouter()
```

Comment:
Just a small spacing detail in here:

```suggestion
  const { query } = useRouter()

```

Reply:
should we add a lint rule to this?

Reply:
For sure! Prettier should've catched this 👀

Reply:
Yea Prettier didn't complain about this one 😆

Reply:
Would you like me to move this constant up above the other constants - to the very top? Instead of alone with this React effect